sf-fx-runtime-java-logger/src/main/java/com/salesforce/functions/jvm/runtime/logger/DefaultLoggingFormatter.java

```diff
@@ -24,16 +26,24 @@ public String format(String loggerName, Level level, String message) {
     String invocationId = MDC.get("function-invocation-id");
     String localDateTimeString = LocalDateTime.now(clock).format(DATE_TIME_FORMATTER);
 
-    return String.format(
-        FORMAT_STRING,
-        localDateTimeString,
-        level.toString(),
-        invocationId,
-        Utils.shortenLoggerName(loggerName, 36),
-        message);
+    String formatString =
+        String.format(
+            FORMAT_STRING,
+            localDateTimeString,
+            level.toString(),
+            invocationId,
+            Utils.shortenLoggerName(loggerName, 36),
+            message);
+    Map<String, char[]> parsedString = Logfmt.parse(formatString.toCharArray());
```

Comment:
Reading the source for [logfmt-java](https://github.com/naaman/logfmt-java/blob/master/src/main/java/com/heroku/logfmt/Logfmt.java), this does note make sense to me. `Logfmt#parse` parses a `logfmt` formatted line and splits the different key/value pairs, putting them in the resulting `Map<String, char[]>`.

The code here builds a pseudo `logfmt` line, parses it with the library and then puts it together again. I'm not sure why it does this, but as-is, this doesn't do anything to ensure the log line is properly formatted (quoting, escaping).

From what I've seen, the linked `logfmt-java` library only does parsing, not generating.

Reply:
Removed logfmt all together, with the dependencies and based my new PR on the expectations of logfmt with the quotations